Library/Homebrew/formula_creator.rb

```diff
@@ -9,11 +9,21 @@ module Homebrew
   #
   # @api private
   class FormulaCreator
-    attr_reader :args, :url, :sha256, :desc, :homepage
-    attr_accessor :name, :version, :tap, :mode, :license
-
-    def initialize(args)
-      @args = args
+    attr_reader :url, :tap, :sha256, :desc, :homepage
```

Comment:
```suggestion
```

if I've read correctly: there can all be removed now.

Alternatively, if you think it makes the code in this file read more nicely with them: could move them below a `private` below to more formally define the public/private APIs and consider adding for e.g. `mode`, `fetch?` etc. Up to you, I don't feel strongly either way.